README.md

```diff
@@ -2,10 +2,56 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+It is the mission of the Node.js Foundation and the TSC to provide a home for 
+openly-governed and community-managed open source projects that are focused on 
+supporting and providing benefit to the entire Node.js ecosystem.
```

Comment:
Is the use of the word "entire" here intentionally setting a very high bar for projects? It's doubtful there are many projects outside of Node core itself that could be interpreted as supporting the *entire* Node.js ecosystem. In fact, I'm hard pressed to come up with a single project that fits this definition.